python_modules/dagster/dagster_tests/definitions_tests/asset_reconciliation_tests/test_asset_reconciliation_sensor.py

```diff
@@ -183,3 +190,43 @@ def repo():
         match=r"build_asset_reconciliation_sensor: Asset '.*' has an AutoMaterializePolicy set",
     ):
         reconciliation_sensor(context)
+
+
+def test_deserialize_asset_reconciliation_cursor_backcompat():
```

Comment:
glad to have this test- it could move to this file with the former backcompat test (which probably needs updating) https://github.com/dagster-io/dagster/blob/master/python_modules/dagster/dagster_tests/definitions_tests/asset_reconciliation_tests/test_asset_daemon_cursor.py

Reply:
good point - moving